docs/sphinx/user_guide/tutorial/gaussSeidel.rst-KEEP

```diff
@@ -50,7 +50,7 @@ index range, has been used in all the other examples. A ``RAJA::ListSegment``
 represents an arbitrary collection of indices, similar to an indirection
 array that is common in unstructured mesh applications. In the example, we
 use two ``RAJA::ListSegment`` objects to hold these two sets of indices. See
-:ref:`index-label` for more information about RAJA segments and index sets.
+:ref:`feat-index-label` for more information about RAJA segments and index sets.
```

Comment:
Maybe remove the `-KEEP` suffix from this file name?

Reply:
It's there since we don't include it in the docs, but we may rework it and include later. The suffix prevents a sphinx warning.